compiler-rt/lib/rtsan/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ else()
                 ${RTSAN_COMMON_RUNTIME_OBJECT_LIBS}
     LINK_FLAGS ${RTSAN_LINK_FLAGS}
     CFLAGS ${RTSAN_CFLAGS}
+    LINK_LIBS ${RTSAN_LINK_LIBS}
```

Comment:
this is static lib, we probably don't need this one?

Reply:
Good with me, removed